SettingsList.py

```diff
@@ -4041,6 +4054,83 @@ def __init__(self, name, gui_text, min, max, default, step=1,
             "hide_when_disabled": True,
         },
     ),
+    Combobox(
+        name           = 'empty_dungeons_mode',
+        gui_text       = 'Empty dungeons',
+        default        = 'none',
+        choices        = {
+            'none':       'Off',
+            'specific':   'Specific Dungeons',
+            'count':      'Count',
+        },
+        gui_tooltip    = '''\
+            Empty dungeons are dungeons guaranted to be barren, and whose rewards
+            are given for free to the player before the beginning of the game.
+
+            'None': No dungeon will be empty. All dungeons may contain useful items
+            and have to be beaten in order to get their rewards.
+            'Specific Dungeons': Choose which specific dungeons will be empty.
+            'Count': Choose how many empty dungeons will be randomly chosen.
+
+            Empty dungeons can't be MQ. Incompatible "MQ Dungeon" settings and
```

Comment:
I would separate the different topics in this paragraph into separate paragraphs.

Reply:
Just to be sure, you mean the paragraph starting line 4075? You think I should split it into four parts:
 - MQ vs empty dungeon,
 - Ignores _Dungeons Have One Major Item_,
 - Dungeon items shuffle,
 - Shuffle songs: dungeon rewards?

Reply:
Yeah. That makes it easier to read imo.